vector/src/main/java/im/vector/app/AppStateHandler.kt

```diff
@@ -117,16 +123,11 @@ class AppStateHandler @Inject constructor(
         return (selectedSpaceDataSource.currentValue?.orNull() as? RoomGroupingMethod.ByLegacyGroup)?.groupSummary?.groupId
     }
 
-    @OnLifecycleEvent(Lifecycle.Event.ON_RESUME)
-    fun entersForeground() {
-    }
-
     @OnLifecycleEvent(Lifecycle.Event.ON_PAUSE)
     fun entersBackground() {
-        compositeDisposable.clear()
```

Comment:
Never disposed then? this change is a bit weird here.
Maybe move the code from `init {}` to `entersForeground()`?

Reply:
Nope I want the invites to be accepted when the app is in background so I can receive calls.
Its just to keep a reference to the disposable so it doesn't get GC

Reply:
So you will restore this line?

Reply:
Thanks!